minvws/nl-kat-coordination
# Feature: improve settings and environment logic and phase out redundant environment keys

### Changes
This improves the environment logic to pass all BOEFJES_* environment variables to the boefjes and allows you to override them with the KATalogus settings. This makes the flow more simple and intuitive.

### Issue link

Closes https://github.com/minvws/nl-kat-coordination/issues/3356

### Demo

-

### QA notes

Please verify the Katalogus and boefjes such as `dns-records` still work properly, creating settings and review the documentation update.

---

### Code Checklist


---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
